beacon-chain/slasher/rpc.go

```diff
@@ -46,11 +46,18 @@ func (s *Service) IsSlashableAttestation(
 		IndexedAttestation: attestation,
 		SigningRoot:        dataRoot,
 	}
+
 	attesterSlashings, err := s.checkSlashableAttestations(ctx, []*slashertypes.IndexedAttestationWrapper{indexedAttWrapper})
 	if err != nil {
 		return nil, status.Errorf(codes.Internal, "Could not check if attestation is slashable: %v", err)
 	}
 	if len(attesterSlashings) == 0 {
+		// Save the attestation record to our database.
```

Comment:
I will  add more color to this comment on why we need to save.
Others might be confused because it is reading like "save attestation only when attester slashing is 0..."